web/src/test/java/com/graphhopper/application/resources/IsochroneResourceTest.java

```diff
@@ -272,6 +272,7 @@ public void requestNotANumber() {
         assertEquals("query param time_limit is not a number.", message);
     }
 
+    @Disabled("block_area is no longer supported and to use custom models we'd need a POST endpoint for isochrones")
```

Comment:
So far we only have `GET` endpoints for `/isochrone` and `/spt` so we cannot use them with custom models. Therefore we don't have an alternative to the current `block_area` feature yet.